### DISCUSSION TITLE
Welcome to massCode Discussions!

### DISCUSSION THREAD

**Discussion by antonreshetov** [massCodeIO/massCode#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andercard0** [comment]
Hi all!
Name is Anderson. musician, I do translations as a hobbie, IT support guy for living and a C# Junior developer.
Working on day by day to become great in my competences.

Just discovered Masscode software and it is being amazing how usefull it is day-to-day, while studying to organize my code examples etc... clean look, great idea and of course, it's free!! I'm trying my best to spread the word of it amongst my collegues. keep the good work Anton!!

**Reply by antonreshetov** [reply]
I am very glad that the app can be a useful tool for every day. Thanks for trying to bring massCode to a larger audience! That's definitely what the app is missing.
